vars/drupal.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+# DRUPAL
+steamengine_drupal_public_path: "{{ steamengine_project_root_path_web }}/{{ steamengine_drupal_public_directory }}"
+steamengine_drupal_persistent_path: "{{ steamengine_project_root_path }}/link"
```

Comment:
Seems to be unused

Reply:
Indeed, fixed in PR https://github.com/STEAMULO/steamulo.steamengine/pull/34